support-files/mysql.spec.sh

```diff
@@ -124,104 +120,42 @@
 %endif
 
 # ----------------------------------------------------------------------------
-# Packager
+# Distribution support
 # ----------------------------------------------------------------------------
-%if %{undefined mysql_packager}
-%define mysql_packager MySQL Build Team <build@mysql.com>
+
+# Disable post build checks for time being.
+BuildConflicts: post-build-checks rpmlint
+
+BuildRequires: gcc-c++ ncurses-devel perl zlib-devel cmake libaio-devel bison flex
+
+%if 0%{?rhel} == 6 || 0%{?rhel} == 7 || 0%{?fedora} == 20 || 0%{?fedora} == 21
+BuildRequires: time
 %endif
 
-# ----------------------------------------------------------------------------
-# Distribution support
-# ----------------------------------------------------------------------------
-%if %{undefined distro_specific}
-%define distro_specific 0
+%if 0%{?suse_version}
+%if 0%{?suse_version} == 1110
+BuildRequires: gdbm-devel gperf openldap2-client procps pwdutils
 %endif
-%if %{distro_specific}
```

Comment:
I think it is technically possible to do it Teemu's way. Still, I'm not convinced it is good:
1) The differences to Oracle's spec file increase, that may make maintenance more difficult.
2) There is no single place any more where you can check which distro versions are supported.
3) The "Distribution:" header field could be quite helpful.
I do not yet see the advantages of dropping this block.